apps/mobile/app/index.tsx

```diff
@@ -3,6 +3,10 @@ import { Redirect } from 'expo-router';
 import { createAuthClient } from '@/modules/auth/auth.client';
 import { configLocalStorage } from '@/modules/config/config.local-storage';
 
+if (__DEV__) {
+  require("./ReactotronConfig");
```

Comment:
Can't import be used instead of require for consistency?

```suggestion
  await import("./ReactotronConfig");
```

Reply:
this is not working probably due to this https://github.com/infinitered/reactotron/issues/1549